ic-os/guestos/envs/prod/BUILD.bazel

```diff
@@ -9,6 +9,11 @@ load("//publish:defs.bzl", "checksum_rule")
 icos_build(
     name = "prod",
     image_deps_func = image_deps,
+    max_file_sizes = {
+        "disk-img.tar.zst": str(450 * 1000 * 1000),  # 419 MB on 2025-03-21
```

Comment:
Why not do the string conversion in the `file_size_check` macro? That way you can specify sizes using numbers which is less error prone.